executor/point_get.go

```diff
@@ -109,14 +113,6 @@ func (e *PointGetExecutor) Next(ctx context.Context, req *chunk.RecordBatch) err
 		if err1 != nil {
 			return err1
 		}
-
-		failpoint.Inject("pointGetRepeatableReadTest", func(val failpoint.Value) {
```

Comment:
If you remove the gofail here, how to make sure the second `get(key)` run after `update` finish?